src/ert/scheduler/slurm_driver.py

```diff
@@ -103,7 +103,7 @@ def __init__(
 
         self._sleep_time_between_cmd_retries = 3
         self._sleep_time_between_kills = 30
-        self._poll_period = 10 if squeue_timeout is None else int(squeue_timeout)
+        self._poll_period = 2 if squeue_timeout is None else int(squeue_timeout)
```

Comment:
It's not really this PR, but why `squeue_timeout` needs to be a string? In other driver we use explicit types of parameters...

Reply:
I think that is legacy from the C-driver. Can be changed yes.